entrypoint.sh

```diff
@@ -39,6 +39,14 @@ if [[ "${INPUT_REPORTER}" = 'github-pr-review' ]]; then
     -level="${INPUT_LEVEL}" \
     -fail-on-error="${INPUT_FAIL_ON_ERROR}" \
     ${INPUT_REVIEWDOG_FLAGS} || reviewdog_exit_val="$?"
+
+  # regenerate black_check_output that command same as non-github-pr-review.
```

Comment:
Maybe the shorter `Re-generate black output. Needed because the output of the '--diff' option can not be used to retrieve the files that black would change` is enough.